Added some corrections and improvements (#67)
hsf-training/hsf-training-cicd-github: _episodes/02-enter-sandman.md

~~~
{: .language-bash}

and then make it executable `chmod +x bash_exit.sh`. Now, try running it with `./bash_exit.sh hello` and `./bash_exit.sh goodbye`.

> ~~~

Reviewer: why not just `echo $?` instead of this more complicated snippet?

Author: I see what you mean. I was trying to have it follow the example before. Would you rather I change it 'echo $?'? 
So something like
code block:
...
and then make it executable 'chmod +x bash_exit.sh'. Let's check the exit code wit `echo $?`.

Author: I believe if we do this we can skip/remove the part where we use the python interpreter.

Reviewer: Checking the status from within python was an important learning point of the previous section, but there's no reason to use it here when we're just playing around in bash

~~~
{: .language-bash}

> and then make it executable `chmod +x bash_exit.sh`. Now, try running it with `./bash_exit.sh hello` and `./bash_exit.sh goodbye` and see what those exit codes are.

Reviewer: ```suggestion
and then make it executable `chmod +x bash_exit.sh`. Now, try running it with `./bash_exit.sh hello` and `./bash_exit.sh goodbye` and see what those exit codes are with `echo $?`.
```

> and then make it an executable `chmod +x bash_exit.sh`.  Let's check the exit code with `echo $?`. See what those exit codes are.

Reviewer: ```suggestion
and then make it executable `chmod +x bash_exit.sh`. Now, try running it with `./bash_exit.sh hello` and `./bash_exit.sh goodbye` and see what those exit codes are with `echo $?`.
```

~~~
{: .language-python}

> and then make it an executable `chmod +x python_exit.py`.  Let's check the exit code with `echo $?`. Déjà vu?

Reviewer 2: I'd leave the statement to run it with `./python_exit.py` here. Otherwise you jump from `chmod` to checking the exit code of `chmod`...

Reviewer 2: Same above with `bash_exit.sh`.

Reviewer: Rolling back

```suggestion
and then make it executable `chmod +x python_exit.py`. Now, try running it with `./python_exit.py hello` and `./python_exit.py goodbye` and see what those exit codes are. Déjà vu?
```
